spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/registry/RdbmsUriRegistry.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.registry;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.sql.DataSource;
+
+import org.springframework.cloud.deployer.resource.registry.UriRegistry;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.util.Assert;
+
+/**
+ * RDBMS implementation of {@link UriRegistry}.
+ *
+ * @author Ilayaperumal Gopinathan
+ * @author Mark Fisher
+ */
+public class RdbmsUriRegistry implements UriRegistry {
+
+    private static final String TABLE_NAME = "URI_APP_REGISTRY";
```

Comment:
the table name should probably just be `URI_REGISTRY` since the "App" part is really a layer over this provided on the dataflow server side